app/models/miq_action.rb

```diff
@@ -829,6 +829,26 @@ def action_ems_refresh(action, rec, inputs)
     end
   end
 
+  def action_container_image_analyze(action, rec, inputs)
+    action_method = "action_container_image_analyze"
```

Comment:
It's been in ruby for a while, at least since 2.0... Maybe even in 1.9.3 and will allow you to avoid a variable assignment